project/Deps.scala

```diff
@@ -46,4 +46,6 @@ object Deps {
 
   // curator service discovery
   val curatorSD = "org.apache.curator" % "curator-x-discovery" % "2.11.0"
+  // kafka
+  val kafka = "org.apache.kafka" % "kafka_2.10" % "0.8.2.1"
```

Comment:
zipkin does not play nice with higher kafka versions. I can switch to scala 2.11 though.